ITEM.tsv

```diff
@@ -398,206 +398,206 @@ ITEM_20150317_000397	$Viper
+ITEM_20150317_000431	$Crossbow 
+ITEM_20150317_000432	$Heavy Dokyu 
+ITEM_20150317_000433	$Heavy Quarrel Bow 
+ITEM_20150317_000434	$Heavy Oak Crossbow 
+ITEM_20150317_000435	$Dunkel Quarrel Bow 
+ITEM_20150317_000436	$Dunkel Cranequine 
+ITEM_20150317_000437	$Dunkel Crossbow 
+ITEM_20150317_000438	$Dunkel Dokyu 
+ITEM_20150317_000439	$Dunkel Oak Crossbow 
+ITEM_20150317_000440	$Arbalest 
+ITEM_20150317_000441	$Fedimian Turret 
+ITEM_20150317_000442	$Bullet Shooter 
+ITEM_20150317_000443	$Black Tip Bow 
+ITEM_20150317_000444	$Spiked Crossbow 
+ITEM_20150317_000445	$Windless 
+ITEM_20150317_000446	$Heavy Crossbow 
+ITEM_20150317_000447	$Though slightly heavy, the magics infused into the bow itself offset this downside. 
+ITEM_20150317_000448	$Wide Crossbow 
+ITEM_20150317_000449	$For those with good aim, this is a good crossbow to deal fatal blows to targets. 
+ITEM_20150317_000450	$Grand Cross 
+ITEM_20150317_000451	$A very popular crossbow among Archers because of its high quality and lack of drawbacks. 
+ITEM_20150317_000452	$Cheminis Crossbow 
+ITEM_20150317_000453	$Weapons based on Alchemy are in great demand since the Great Plant Cataclysm. 
+ITEM_20150317_000454	$Medina Crossbow 
+ITEM_20150317_000455	$This weapon is specialized in taking down well-equipped enemies. 
+ITEM_20150317_000456	$Regina Crossbow 
+ITEM_20150317_000457	$Not good enough to be considered the "Lord of Crossbows," but the title of "Queen" seems fair. 
+ITEM_20150317_000458	$Chaser 
+ITEM_20150317_000459	$Frequently used by archers operating in areas with poor visibility or strong winds. 
+ITEM_20150317_000460	$Imperni Crossbow 
+ITEM_20150317_000461	$Rumors say that black market weapons are intentionally made to be expensive to maintain. 
+ITEM_20150317_000462	$Grave Crossbow 
+ITEM_20150317_000463	$This weapon is recommended if you intend to permanently put the target to sleep. 
+ITEM_20150317_000464	$Smith Crossbow 
+ITEM_20150317_000465	$This weapon will only deal as much damage as you expect it to every time you pull the trigger. 
+ITEM_20150317_000466	$Klavis Crossbow 
+ITEM_20150317_000467	$You could be the first person to draw out the true potential of this weapon. 
+ITEM_20150317_000468	$Tresh Crossbow 
+ITEM_20150317_000469	$Sestas Crossbow 
+ITEM_20150317_000470	$Drat Crossbow 
+ITEM_20150317_000471	$Aston Crossbow 
+ITEM_20150317_000472	$Devi Crossbow 
+ITEM_20150317_000473	$Prima Crossbow 
+ITEM_20150317_000474	$Shooting Star 
+ITEM_20150317_000475	$Previously only archers who completed the training in 'The Tower of the Stars' could use this weapon. 
+ITEM_20150317_000476	$Katyn Blaster 
+ITEM_20150317_000477	$The Ranger Master developed this weapon based on research data on the monsters in the Katyn Forest after the Great Plant Cataclysm. However, he recommends that no one uses it. 
+ITEM_20150317_000478	$Grajus 
+ITEM_20150317_000479	$Can large monster be hunted using only a one-handed crossbow? This is the answer. 
```

Comment:
The original was correct, but if this is going to be changed it needs to be either:
"Can a large monster"
or
"Can large monsters"